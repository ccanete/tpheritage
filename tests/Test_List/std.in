R test1 1 2 2 4
L test2 2 4 5 6
S test 1 2 2 4
LIST
EXIT