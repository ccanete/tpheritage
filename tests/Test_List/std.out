OK
# The file test has been loaded.
OK
# Rectangle test1 of coordinates (1, 2) and (2, 4) has been created.
OK
# Line test2 passing by (2, 4) and (5, 6) has been created.
OK
# Selection test of coordinates (1, 2) and (2, 4) has been created. 
R test1 1 2 2 4
L test2 2 4 5 6

# The program is being closed.